File: william/1FY901/projekt/pendulum.py
import numpy as np

def pendulum(t, y, A, f):
    
    g = 9.82
    
    phi = y[0]
    phi_dot = y[1]
    
    phi_ddot = -np.sin(phi) * (A * t**2 * np.cos(f * t) + g)
    
    dydt = [phi_dot, phi_ddot]
    return dydt

File: yoda/cidmv/1FY901/projekt/pendulum.m
function dydt = pendulum(t, y, A, f)
    g = 9.82;   
    l = 1;
    phi = y(1);
    phi_dot = y(2);
    
    phi_ddot = -(sin(phi) * (A * f^2 * cos(f*t) + g*l)) / l;
    
    dydt = [phi_dot; phi_ddot];
end
